modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -200,13 +205,18 @@ Otherwise it is matched as a case-sensitive string.
 | List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. Possible values are: +
 
 - `PacketDrop`: enable the packets drop flows logging feature. This feature requires mounting
-the kernel debug filesystem, so the eBPF pod has to run as privileged.
+the kernel debug filesystem, so the eBPF agent pods have to run as privileged.
 If the `spec.agent.ebpf.privileged` parameter is not set, an error is reported. +
 
 - `DNSTracking`: enable the DNS tracking feature. +
 
 - `FlowRTT`: enable flow latency (sRTT) extraction in the eBPF agent from TCP traffic. +
 
+- `NetworkEvents`: enable the Network events monitoring feature, such as correlating flows and network policies.
+This feature requires mounting the kernel debug filesystem, so the eBPF agent pods have to run as privileged.
+It requires using the OVN-Kubernetes network plugin with the Observability feature.
+IMPORTANT: this feature is available as a Developer Preview. +
```

Comment:
```suggestion
IMPORTANT: This feature is available as a Developer Preview. +
```

Reply:
I also think that `+` shouldn't be there. Good point though about this one.